docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+     "text": [
+      "Sentence:\tEu rejects German call to boycott British lamb.\n",
+      "Given words:\t['Eu', 'rejects', 'German', 'call', 'to', 'boycott', 'British', 'lamb', '.']\n",
+      "Subwords:\t['E', 'u', 'rejects', 'German', 'call', 'to', 'boycott', 'British', 'la', 'mb', '.']\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Sentence:\\t' + sentences[0]) \n",
+    "print('Given words:\\t' + str(given_words[0])) \n",
+    "print('Subwords:\\t' + str(tokens)) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16eccd57",
+   "metadata": {},
+   "source": [
+    "The word `lamb` is tokenized into two subwords `la` and `mb`. In this case, we assign the average predictive probabilities of the two subwords to the token. Alternatively, we can take the weighted average, such that the weight for each predictive probability is proportional to the length of its corresponding subword. This is to ensure that longer subwords have heavier weights on the average predictive probabilities, although the benefits are insignificant for most datasets. \n",
+    "\n",
+    "Each tokenizer tokenizes sentences differently. In some rare cases, a subword may overlap two tokens, resulting in a misalignment in tokenization. For example, consider the following tokenization: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "d0565e85",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sentence:\tMassachusetts Institute of Technology (MIT)\n",
+      "Given words:\t['Massachusetts', 'Institute', 'of', 'Technology', '(', 'MIT', ')']\n",
+      "Subwords:\t['Massachusetts', 'Institute', 'of', 'Technology', '(M', 'IT', ')']\n"
+     ]
+    }
+   ],
+   "source": [
+    "demo_sentence = 'Massachusetts Institute of Technology (MIT)' \n",
+    "demo_given_words = ['Massachusetts', 'Institute', 'of', 'Technology', '(', 'MIT', ')'] \n",
+    "demo_subwords = ['Massachusetts', 'Institute', 'of', 'Technology', '(M', 'IT', ')'] \n",
+    "\n",
+    "print('Sentence:\\t' + demo_sentence) \n",
+    "print('Given words:\\t' + str(demo_given_words)) \n",
+    "print('Subwords:\\t' + str(demo_subwords)) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a794f3d0",
+   "metadata": {},
+   "source": [
+    "In this case, we assign the predictive probabilities of `(M` to `(`, and the average predictive probabilities of `(M` and `IT` to `MIT`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "f3656f29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred_probs_and_labels = [get_pred_probs_and_labels(sentence_probs[i], \n",
+    "                                                   sentence_tokens[i], \n",
+    "                                                   given_words[i], \n",
+    "                                                   given_labels[i]) for i in range(len(sentences))] "
+   ]
+  },
```

Comment:
This is the most crucial step for the user to exactly right, so we should print out a few example sentences with:
- their indivdual tokens
- the class-label for each token
- the pred_probs for each token

with some text in markdown cell discussing these format